### ISSUE TITLE
Verify that puppeteer generates accessible PDFs

### ISSUE THREAD

**Issue by fniessink** [ICTU/quality-time#8053]
This should be fixed in Quality-time v5.8.0 with puppeteer 22.0.0 (that contains https://github.com/puppeteer/puppeteer/pull/11778).

**Comment by Sebastiaan127001** [comment]
According to [PAC](https://pdfua.foundation/en/pac-download/), a widely recognised tool. This generated PDF is not compliciant to WCAG and not accessible. See attached detailed report
[Quality-time-report-example-report-2024-02-08T14_04_31_PAC_UA_Report.pdf](https://github.com/ICTU/quality-time/files/14345123/Quality-time-report-example-report-2024-02-08T14_04_31_PAC_UA_Report.pdf)
.

![image](https://github.com/ICTU/quality-time/assets/95095453/ec91cf02-9a76-4441-8ba3-53df5e65cae0)

**Comment by Sebastiaan127001** [comment]
For comparison purposes, here is a report generated with a Quality-time PDF report from a year ago.
![image](https://github.com/ICTU/quality-time/assets/95095453/69e2eb09-ac4a-42cc-bbdb-11bf92b2c307)
[Quality-time-GIR2-2023-06-14_PAC_UA_Report.pdf](https://github.com/ICTU/quality-time/files/14345313/Quality-time-GIR2-2023-06-14_PAC_UA_Report.pdf)

**Comment by Sebastiaan127001** [comment]
[The puppeteer feature](https://github.com/puppeteer/puppeteer/pull/11778) states that it should be 'more accessible' but not that it is PDF/UA WCAG-compliant, which is truly accessible, according to standards.

**Comment by fniessink** [comment]
Thanks for checking.
